decompile/disasm.ts

```diff
@@ -90,15 +90,14 @@ export class Dissasembler {
         this.#emit(".out", reg);
       }
     });
-    const subOffset = this.subs.length;
```

Comment:
A subOffset is still needed when compiling multiple behaviors, see the example I added.

Reply:
This now passes both the test, but I am not happy with the `isBlueprint` boolean.

Reply:
Ok, found a way to not need isBlueprint, and not need to have the partial list of subs in the Dissasembler object